accelerator/config.h

```diff
@@ -65,6 +65,8 @@ extern "C" {
 #define HEALTH_TRACK_PERIOD 1800              // Check every half hour in default
 #define RESULT_SET_LIMIT 100  // The maximun returned transaction object number when querying transaction object by tag
 
+bool cli_mode;
```

Comment:
Can we avoid such global variable(s)?

Reply:
I'll move both `cli_mode` and `quiet_mode` variable to `ta_config_t` since these two variables seem to represent similar like `proxy_passthrough` and `gtta` in `ta_config_t` structure (both of them represent the command line input)
cc @HowJMay @YingHan-Chen

Reply:
Maybe we could use bitwise manner to store these information in a `long` object?